specs/core/1_custody-game.md

```diff
@@ -180,8 +184,12 @@ Add the following fields to the end of the specified container objects. Fields w
 #### `Validator`
 
 ```python
-    'custody_reveal_index': 'uint64',
-    'max_reveal_lateness': 'uint64',
+    '''
+    latest_custody_reveal_period is initialised to the custody period
+    (of the particular validator) *AFTER* which the validator was activated.
+    = get_validators_current_custody_reveal_period(...) + 1
+    '''
+    'latest_custody_reveal_period': 'uint64',
```

Comment:
So the first subkey that the validator will be revealing is from the period that is _two_ periods ahead of the current one? Not sure I see the logic here. You sure we don't want `- 1`?

And if it is -1, then it seems better to avoid negative numbers and keep track of `next_custody_reveal_period` instead of `latest_custody_reveal_period`.